crates/icp-cli/src/commands/canister/settings/mod.rs

```diff
@@ -7,7 +7,10 @@ pub(crate) mod update;
 #[derive(Subcommand, Debug)]
 #[allow(clippy::large_enum_variant)]
 pub(crate) enum Command {
-    /// Display a canister's settings
+    /// Show the status of a canister. By default this queries the
```

Comment:
This whole paragraph is shown in `icp canister settings help`. Double linebreak after the first sentence prevents that.